forms/LiteralField.php

```diff
@@ -1,76 +1,103 @@
 <?php
+
 /**
  * This field lets you put an arbitrary piece of HTML into your forms.
- * 
- * <b>Usage</b>
- * 
+ *
  * <code>
  * new LiteralField (
  *    $name = "literalfield",
  *    $content = '<b>some bold text</b> and <a href="http://silverstripe.com">a link</a>'
  * )
  * </code>
- * 
+ *
  * @package forms
  * @subpackage fields-dataless
  */
 class LiteralField extends DatalessField {
-	
 	/**
-	 * @var string $content
+	 * @var string|FormField
```

Comment:
I think we are just meant to use string. If you use a FormField as a value, the field internally relies on the `__toString` to cast it safely to HTML safe text. Should we thus limit our expectations to strings only?